SpotifyApp/SpotifyApp/SpotifyTableViewController.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  AlbumTableViewController.swift
+//  SpotifyApp
+//
+//  Created by Eashir Arafat on 10/28/16.
+//  Copyright © 2016 Evan. All rights reserved.
+//
+
+import UIKit
+
+class SpotifyTableViewController: UITableViewController {
+   
+    internal let AlbumJSONFileName: String = "SpotifyAlbums.json"
+    internal var Albums: [Album] = []
+    
+    internal let AlbumEndpoint: String = "https://api.spotify.com/v1/search?q=blue&type=album&limit=50"
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.getAlbums(from: AlbumEndpoint)
+    }
+    
+    func getAlbums(from apiEndpoint: String) { // <<< returns Void
```

Comment:
This method belongs in one of your classes, not in your view controller, as per M-V-C.